lighthouse-core/gather/gatherers/seo/font-size.js

```diff
@@ -182,7 +182,7 @@ function getEffectiveFontRule({inlineStyle, matchedCSSRules, inherited}) {
  * @returns {number}
  */
 function getNodeTextLength(node) {
-  return !node.nodeValue ? 0 : node.nodeValue.trim().length;
+  return !node.nodeValue ? 0 : Array.from(node.nodeValue.trim()).length;
```

Comment:
Can we get a comment mentioning this PR or explaining why this is here. e.g.
```
// Use Array.from in order to more accurately count unicode characters. See: #6973
```
Maybe even add a `TODO` for @patrickhulce's comment on future rabbit hole exploring.